capstone-app/src/app/app.component.html

```diff
@@ -1,8 +1,15 @@
-<h1>{{title}}</h1>
+<head> <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet"></head>
+<h1 class="title">
+  <span id='G'>G</span>
```

Comment:
Use a consistent style for single-quotes/double-quotes. Typically, we use double quotes for HTML and single quotes for TypeScript.